### PR TITLE
Implement repeated-prefix, Fix #844

### LINKED ISSUE #844: Feature request: repeated-prefix
Much like repeated-time, but for prefix, so that we can apply conditional styling when the same person is sending multiple messages in a row without interruption by others (for example to hide the prefix).

### PR DESCRIPTION
**Pull request by Vaesper**
Fix #844 

Related to #605 , but not the same as it allows conditional styling in CSS (ie. one can suppress duplicate prefixes in desktop layout but show them in mobile layout).

### CODE REVIEW

**Comment by Vaesper** [comment]
Should conflict with #843 so if either is merged I will have to manually rebase the other.

File: js/models.js [hunk 1]
```diff
@@ -365,12 +365,17 @@ models.service('models', ['$rootScope', '$filter', 'bufferResume', function($roo
...
+            prefixtext += prefix[pti].text;
+        }
+
         var rtext = "";
         for (var i = 0; i < content.length; ++i) {
             rtext += content[i].text;
         }
 
-       return {
+        return {
```
Comment (lorenzhs) [inline js/models.js]: ehk! Sorry for the pedantry, can you amend this out?

Comment (Vaesper) [inline js/models.js]: Why? It's aligned properly now!

Comment (lorenzhs) [inline js/models.js]: Oh it was wrong before. Sorry, didn't catch that. Thanks for fixing in that case :)

**Comment by lorenzhs** [comment]
Thanks!

**Pull request merged**